internal/config/config.go

```diff
@@ -72,12 +79,28 @@ func Read(path string, cfg interface{}) error {
 	return err
 }
 
-// GetActualValue returns the environment variable value if the val has prefix and suffix "_", otherwise the val will directly return.
-func GetActualValue(val string) string {
-	if checkPrefixAndSuffix(val, "_", "_") {
-		return os.ExpandEnv(os.Getenv(strings.TrimPrefix(strings.TrimSuffix(val, "_"), "_")))
+// GetActualValue returns the environment variable value if the val has prefix and suffix "_", 
+// if actual value start with file://{path} the return value will read from file
+// otherwise the val will directly return.
+func GetActualValue(val string) (res string) {
+	if checkPrefixAndSuffix(val, envSymbol, envSymbol) {
+		val = strings.TrimPrefix(strings.TrimSuffix(val, envSymbol), envSymbol)
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blueviolet.svg?style=flat-square)
hmm...
`os.Getenv` is not needed in new codes? @kpango

Reply:
yes, Getenv will resolved in ExpandEnv function

Reply:
I think `ExpandEnv` doesn't resolve as we expected.
If we call `GetActualValue("_SECRET_")`, it will return just "SECRET" string. :thinking:
Or, should we pass the argument as `GetActualValue("_$SECRET_")` ?
https://golang.org/pkg/os/#ExpandEnv

Reply:
hmm how about if I add the $ prefix in env if statement

Reply:
Okay! That's the simple answer. :+1: